.gitpod.yml

```diff
@@ -0,0 +1,11 @@
+# Commands to start on workspace startup
+tasks:
+  - init: go build -o grid-intensity
+    command: |
+      export PATH="$PATH:."
```

Comment:
Would you please add a one line comment to make it easier to understand why this is added? 

I support the decision, and I get why you would do this for easy of use, and DX but if you're not using the `./` prefix, I think it would help for folks who aren't used to messing around with $PATH all that much.

Reply:
@mrchrisadams Yes definitely. They are a bit longer but I added some comments for the bash magic. Does it look OK?

Reply:
Brilliant, thank you @rossf7. This will be helpful for me, and I'll also look to include it in the docs because others might benefit from it as well.

Reply:
Thanks @fershad glad it's helpful and will be good to have it in the docs.